tasks/taxon_id/contamination/task_krona.wdl

```diff
@@ -4,32 +4,32 @@ task krona {
   input {
     File kraken2_report
     String samplename
-    String docker = "us-docker.pkg.dev/general-theiagen/biocontainers/krona:2.7.1--pl526_5"
+    String docker = "us-docker.pkg.dev/general-theiagen/staphb/krona:2.8.1"
     Int memory = 8
-    Int cpu = 4
+    Int cpu = 2
     Int disk_size = 100
   }
   command <<<
     # Get VERSION
-    ktImportTaxonomy 2>&1 | sed -n '/KronaTools /p' | sed 's/^.*KronaTools //; s/ - ktImportTaxonomy.*//' | tee VERSION
+    ktImportText | grep "KronaTools" | cut -d' ' -f3 | tee VERSION
 
-    # Get taxonomy file 
-    ktUpdateTaxonomy.sh taxonomy
+    # run KrakenTools kreport2krona.py to enable viral compatibility
+    python3 /KrakenTools/kreport2krona.py -r ~{kraken2_report} -o ~{samplename}_krona.txt
 
     # Run krona with taxonomy on krakren report
-    ktImportTaxonomy -o ~{samplename}_krona.html ~{kraken2_report} -tax taxonomy
```

Comment:
For reference purposes: https://github.com/marbl/Krona/wiki/Importing-text-and-XML-data